plugins/modules/elastic_expedient_security_rules.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python
+# Copyright 2021 Expedient
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+DOCUMENTATION='''
+
+module: elastic_expedient_security_rules
+
+author: Ian Scott
+
+short_description: Update Security rules to the Expedient Standard
+
+description: 
+  - Update Security rules to the Expedient Standard
+
+requirements:
+  - python3
+
+options:
+      host: ECE Host or Deployment Host
+      port: ECE Port or Deployment Port
+      username: ECE Username or Deployment Username
+      password: ECE Password or Deployment Password
+      deployment_info: (when using ECE host:port and credentials)
+        deployment_id: ECE Deployment ID
+        deployment_name: ECE Deployment Name
+        resource_type: kibana
+        ref_id: REF ID for kibana cluster, most likely main-kibana
+        version: Deployment Kibana Version
```

Comment:
can you update this to follow the same schema as the other `DOCUMENTATION` strings?  it's used to generate the automatic help files in `ansible-help` as well as for the .rst file generation for tower hub

Reply:
take a look at the one here - https://github.com/Expedient/ansible-collection-vcloud/blob/main/plugins/modules/vcloud_datacenter_group.py there are some additional details to include underneath each option that clarify what each of them does, as well as some other metadata to include like the name of the module, descriptions, etc.

full documentation on how the `DOCUMENTATION` string works is here - https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html